test/remote/gateways/remote_ct_payment_certification_test.rb

```diff
@@ -0,0 +1,246 @@
+require 'test_helper'
+
+class RemoteCtPaymentCertificationTest < Test::Unit::TestCase
+  def setup
+    @gateway = CtPaymentGateway.new(fixtures(:ct_payment))
+
+    @amount = 100
+    @declined_card = credit_card('4502244713161718')
+    @options = {
+      billing_address: address,
+      description: 'Store Purchase',
+      merchant_terminal_number: '13366',
+      order_id: generate_unique_id[0,11]
+    }
+  end
+
+  def test1
```

Comment:
~Do these test numbers correlate to ones that are supposed to be in the docs? Why are they all just `test<num>`?~ Okay, I get it; file name says yes.